torch/nn/modules/container.py

```diff
@@ -135,6 +135,46 @@ def __add__(self, other) -> 'Sequential':
                              'of Sequential class, but {} is given.'.format(
                                  str(type(other))))
 
+    def __iadd__(self, other: 'Sequential') -> 'Sequential':
+        if isinstance(other, __class__):
+            offset = len(self)
+            for i, module in enumerate(other):
+                self.add_module(str(i+offset), module)
+            return self
+        else:
+            raise ValueError(f"unsupported operand type(s) for +=: {type(self)} and {type(other)}")
+
+    def __mul__(self, other: int) -> 'Sequential':
+        if not isinstance(other, int):
+            raise TypeError(f"unsupported operand type(s) for *: {type(self)} and {type(other)}")
+        elif (other <= 0):
+            raise ValueError(f"Non-positive multiplication factor {other} for {type(self)}")
+        else:
+            combined = Sequential()
+            offset = 0
+            for _ in range(other):
+                for module in self:
+                    combined.add_module(str(offset), module)
+                    offset += 1
+            return combined
+
+    def __rmul__(self, other: int) -> 'Sequential':
+        return self.mul(other)
```

Comment:
Shouldn't this be `self.__mul__(other)` or `self * other` ?

Reply:
Yup, my bad!